sources/packages/web/src/views/aest/institution/InstitutionLocationEdit.vue

```diff
@@ -36,18 +36,21 @@ export default defineComponent({
   setup(props) {
     const initialData = ref({} as InstitutionLocationEdit);
     const snackBar = useSnackBar();
+    const { excludeExtraneousValues } = useFormioUtils();
     const router = useRouter();
-    const updateInstitutionLocation = async (
-      data: InstitutionLocationAPIInDTO,
-    ) => {
+    const updateInstitutionLocation = async (data: unknown) => {
```

Comment:
Should we declare unknown here where the submit/emit happens?

![image](https://github.com/bcgov/SIMS/assets/54600590/9922f174-5827-41c3-b28f-63781537871e)

Reply:
At this moment I am not sure what would be clear for a developer in the future.
Maybe the ideal scenario would have a dedicated model for every form.io and then have it properly mapped.
This form.io, for instance, I hope that it would be converted to a regular Vuetify someday.
For now, I am changing it to unknow.